os/arch/arm/include/syscall.h

```diff
@@ -68,6 +68,8 @@
 #include <arch/armv7-r/syscall.h>
 #elif defined(CONFIG_ARCH_CORTEXM33) || defined(CONFIG_ARCH_CORTEXM55)
 #include <arch/armv8-m/syscall.h>
+#elif defined(CONFIG_ARCH_CORTEXA9)
```

Comment:
Since we added Cortex A32 in Kconfig, we might need to add it here also. Please check similar cases.